src/loader.js

```diff
@@ -5,155 +5,198 @@ let Promise = require('promise');
 let _ = require('lodash');
 let TopoSort = require('topo-sort');
 
-// We use this closure to create a .create() method which
-// returns a graphViz file for a given component directory
-function generateGraph(dir, vComp) {
-  let tsort = new TopoSort();
-
-  for (let x of _.keys(dir)) {
-    tsort.add(x, dir[x].requires || []);
+/** Check if a value is a flat value or a component definition */
+let isComponent = (def) => {
```

Comment:
Yep, should've done this from the beginning.  Any reason to not make it `function isComponent (def) {...}`?  I think that reads much cleaner